src/intercooler.js

```diff
@@ -1755,13 +1760,14 @@ var Intercooler = Intercooler || (function() {
       }
     }
 
-    function snapshotForHistory(newUrl) {
+    function snapshotForHistory(newUrl, oldTitle) {
       var t = getTargetForHistory($('body'));
       triggerEvent(t, "beforeHistorySnapshot.ic", [t]);
       _snapshot = {
         newUrl: newUrl,
         oldHtml: t.html(),
-        yOffset: window.pageYOffset
+        yOffset: window.pageYOffset,
+        title: oldTitle
```

Comment:
Let's call this field `oldTitle`, to be consistent with `oldHtml` -- in my mind they really go together, and thinking of them as a pair helps me to follow the flow of data through the snapshot code.

Combined with the comment I made about the variable above, this line would simply be `oldTitle: document.title`.

Reply:
Fixed the name of the variable to oldTitle. See explanation above why it cannot be `document.title`.